src/components/pages/Registration/InstructorFlow_Step2.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState, useEffect } from 'react';
+import '../../../styles/registration.less';
+import InstructorFormSchema from './InstructorFormSchema';
+import * as yup from 'yup';
+
+const initialValues = {
+  name: '',
+  email: '',
+  location: '',
+  phone: '',
+  education: '',
+  tech: '',
+  notes: '',
+};
+
+const initialErrors = {
+  name: '',
+  email: '',
+  location: '',
+  phone: '',
+  education: '',
+};
+
+const initialSaveDisabled = true;
+
+const InstrRegForm = () => {
+  const [formValues, setFormValues] = useState(initialValues);
+  const [formErrors, setFormErrors] = useState(initialErrors);
+  const [disabled, setDisabled] = useState(initialSaveDisabled);
+
+  const validate = (name, value) => {
+    yup
+      .reach(InstructorFormSchema, name)
+      .validate(value)
+      .then(() => setFormErrors({ ...formErrors, [name]: '' }))
+      .catch(err => setFormErrors({ ...formErrors, [name]: err.errors[0] }));
+  };
+
+  const inputChange = (name, value) => {
+    validate(name, value);
+    setFormValues({
+      ...formValues,
+      [name]: value,
+    });
+  };
+
+  const formSave = () => {
+    // TODO
+  };
+
+  const onSubmit = evt => {
+    evt.preventDefault();
+    formSave();
+    setFormValues(initialValues);
+  };
+
+  const onChange = evt => {
+    const { name, value } = evt.target;
+    inputChange(name, value);
+  };
+
+  useEffect(() => {
+    InstructorFormSchema.isValid(formValues).then(valid => setDisabled(!valid));
+  }, [formValues]);
+
+  return (
+    <div>
```

Comment:
Great start to have a working page as you build out the form! The structure of this can be adjusted in the future to implement the formatting of the design.  The outermost div could have the 'reg-content-container' class and the inner divs could be broken up according to the format. (one div to hold the progress bar at the top, one div to hold the form, the form title can live within the form container )